packages/web/src/scss/components/Pagination/README.md

```diff
@@ -0,0 +1,181 @@
+# Pagination
+
+First item selected:
+
+```html
+<nav role="navigation" aria-label="Pagination Navigation" class="Pagination mb-800">
+  <ul class="Pagination__items">
+    <li>
+      <a href="#" class="Button Button--secondary Button--square Button--disabled">
+        <svg width="24" height="24" aria-hidden="true">
+          <use xlink:href="/icons/svg/sprite.svg#chevron-left"></use>
```

Comment:
I think `<use>` is not a pair tag. https://developer.mozilla.org/en-US/docs/Web/SVG/Element/use
```suggestion
          <use xlink:href="/icons/svg/sprite.svg#chevron-left" />
```

Reply:
👍